src/attestedclaim/AttestedClaim.ts

```diff
@@ -20,7 +20,14 @@ import IRequestForAttestation from '../types/RequestForAttestation'
 
 export default class AttestedClaim implements IAttestedClaim {
   /**
-   * Creates a new instance of this Attestation class from the given interface.
+   * @description (STATIC) Creates a new [[AttestedClaim]] instance from the given interface.
+   * @param obj The base object from which to create the attested claim.
+   * @returns A new attested claim.
+   * @example
+   * ```javascript
+   * // if cloneDeep is a utility function that deep clones objects
+   * const attestedClaimCopy = AttestedClaim.fromObject(cloneDeep(attestedClaim));
+   * ```
    */
   public static fromObject(obj: IAttestedClaim): AttestedClaim {
     const newAttestedClaim: AttestedClaim = Object.create(
```

Comment:
@tjwelde Minor question: what's the use of this method, why was it created?   

(I'm not happy with the example I've provided, not to mention that it relies on a lodash utility function).